src/store/userLists/operations/createUserListOperation.js

```diff
@@ -0,0 +1,39 @@
+import { createLogic } from 'redux-logic'
+
+import { openNotification } from 'Utils/'
+import { sessionIdSelector } from 'Store/auth/selectors'
+import { CREATE_USER_LIST } from '../types'
+import { createUserListStart, createUserListFinish, getUserLists } from '../actions'
+
+export const createUserListOperation = createLogic({
+  type: CREATE_USER_LIST,
+  latest: true,
+  async process({ action, axios, getState }, dispatch, done) {
+    const { values, cb } = action
+    const { name, description } = values
+    const sessionId = sessionIdSelector(getState())
+    try {
+      dispatch(createUserListStart())
+      await axios({
+        method: 'post',
+        url: '/list',
+        data: {
+          name,
+          description
+        },
+        params: {
+          session_id: sessionId
+        }
+      })
+      cb()
+    } catch (error) {
+      openNotification({
+        message: error.message,
+        type: 'error'
+      })
+    }
+    dispatch(createUserListFinish())
```

Comment:
are you sure this action need to be called outside `try` block?

Reply:
yes, I need to set `false` for loading state

Reply:
need to be discussed

Reply:
✅